vimiv/api/settings.py

```diff
@@ -475,6 +475,21 @@ class thumbnail:  # pylint: disable=invalid-name
         True,
         desc="Save new thumbnails to disk in the shared icon cache for later use",
     )
+    load_behind = IntSetting(
+        "thumbnail.load_behind",
+        -1,
+        desc="Max thumbnails to render behind the selected one. -1 is no limit",
+    )
+    load_ahead = IntSetting(
+        "thumbnail.load_ahead",
+        -1,
+        desc="Max thumbnails to render ahead of the selected one. -1 is no limit",
+    )
+    unload_threshold = IntSetting(
+        "thumbnail.unload_threshold",
+        -1,
+        desc="Don't unload thumbnails unless more are loaded than this. -1 is no limit",
```

Comment:
Maybe we can somehow specify that they are of course unloaded when the filelist changes? Something along
"Don't unload thumbnails in the current selection unless ... ". Selection is probably not the best word, maybe you have a better one in mind :blush:

Reply:
I'm going for something like... "Don't unload currently rendered thumbnails unless more are loaded than this. -1 is no limit" ?

Reply:
Nice, maybe even "unless above this threshold"?

Reply:
I leaned to something slightly more verbose: `Don't unload currently rendered thumbnails unless **the number loaded is** above this threshold.` Can do the other, though. Or just do 'the number is', since it will refer back to 'currently rendered thumbnails'.

Reply:
Sounds perfect to me, thanks!